be/src/storage/olap_common.h

```diff
@@ -101,6 +101,14 @@ enum RangeCondition {
     LE = 3, // less or equal
 };
 
+enum MaterializeType {
```

Comment:
Why use materialize?

Reply:
> Why use materialize?

Define MaterializeType to get materialize_converter to create materialized view. If a new materialization function is available in the future, we can extend the MaterializeType to get new materialize_converter